styles/_theme.scss

```diff
@@ -659,3 +397,26 @@ blockquote .source {
   padding: 0;
   list-style-position: inside;
 }
+
+.usa-prose {
+  .usa-collection__item {
+    border-top: none;
+  }
+}
+
+h1 {
+  font-family: Source Sans Pro Web, Helvetica Neue, Helvetica, Roboto, Arial, sans-serif;
+}
+
+.usa-intro, .usa-prose>h1+p {
+  max-width: 88ex;
+  font-family: Source Sans Pro Web, Helvetica Neue, Helvetica, Roboto, Arial, sans-serif;
+  font-size: 1.46rem;
+  font-weight: 400;
+  line-height: 1.6;
+}
+
+.usa-icon {
```

Comment:
this is changing all icons everywhere? I don't understand overriding USWDS classes globally. what do we gain for this?

Reply:
Yes, they are not aligned vertically with text so this fixes that.  Okay with removing it or a different approach. Please let me know.